README.md

```diff
@@ -91,6 +91,39 @@ app.get('/', async function (req, reply) {
 app.listen(3000)
 ```
 
+### persistedQueries support
+
```

Comment:
I would add a quick explanation about what persisted queries are and why they're useful

Reply:
Yep, that'd be useful. Done.